.cirrus.yml

```diff
@@ -796,6 +801,51 @@ podman_machine_windows_task:
     main_script: ".\\repo\\contrib\\cirrus\\win-podman-machine-main.ps1"
 
 
+podman_machine_mac_task:
+    name: *std_name_fmt
+    alias: podman_machine_mac
+    only_if: *no_rhel_release
+    depends_on:
+      - build_aarch64
```

Comment:
This makes for a weird-looking graph. How necessary is it really?

Reply:
Hmmm, ahh yes, it only needs to depend on `osx_alt_build`.  Fix staged.